src/mlx90632.c

```diff
@@ -449,4 +471,145 @@ int32_t mlx90632_init(void)
 
     return 0;
 }
+
+int32_t mlx90632_addressed_reset(void)
+{
+    int32_t ret;
+
+    ret = mlx90632_i2c_write(0x3005, MLX90632_RESET_CMD);
+    if (ret < 0)
+        return ret;
+
+    usleep(150, 200);
+
+    return 0;
+}
+
+
+/** Reads the refresh rate and calculates the time needed for a single measurment from the EEPROM settings.
+ *
+ * @param[in] meas Measurement to read the frefresh rate for
+ *
+ * @retval >= Refresh time in ms
```

Comment:
Use same as you have elsewhere

```suggestion
 * @retval >=0 Refresh time in ms
```